src/vivarium_csu_alzheimers/model_specifications/model_spec.yaml

```diff
@@ -3,6 +3,7 @@ components:
         population:
```

Comment:
@stevebachmeier I would just focus on the components in the prs and we can worry about the config values below when we start the profiling adventure.